lib/net/nforceif/src/driver.c

```diff
@@ -117,7 +117,7 @@ low_level_init(struct netif *netif)
   if (netif->mld_mac_filter != NULL) {
     ip6_addr_t ip6_allnodes_ll;
     ip6_addr_set_allnodes_linklocal(&ip6_allnodes_ll);
-    netif->mld_mac_filter(netif, &ip6_allnodes_ll, MLD6_ADD_MAC_FILTER);
+    netif->mld_mac_filter(netif, &ip6_allnodes_ll, NETIF_ADD_MAC_FILTER);
```

Comment:
http://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=6b1e1af3d19c28f26519460531f28b23d4f6e1f0 combined the filter actions for IGMP and MLD6. This adapts the code to use the new constant name, no behavioral change should result from this.